maint_tools/test_docstrings.py

```diff
@@ -16,7 +16,6 @@
     "BaggingRegressor",
     "BernoulliNB",
```

Comment:
I think this PR should be split in 2 PR to make them easier to review. 

One about naive Bayes models and the other about the `Binarizer` model.

For the one about `BernoulliNB` should probably updated all the `NB` models together.

Reply:
Oh, I see, thank you! Yes I have a branch for BrenoulliNB, too. Did not mean to add that to this PR. Sorry about that. It's still a WIP

Reply:
I'll work on separating these two things

Reply:
I've removed all the changes related to BernoulliNB.